requirements.txt

```diff
@@ -0,0 +1,10 @@
+Cython>=0.28,<1.0; python_version == '3.7'
+PyStemmer==1.3.0
+bblfsh==2.12.7
+modelforge==0.12.1
+numpy==1.16.2
+humanize==0.5.1
+pygments==2.3.1
+keras==2.2.4
+scikit-learn==0.20.3
+tqdm==4.31.1
```

Comment:
modelforge should require it already (but please check)

Reply:
@Guillemdb did you check?

Reply:
ok, modelforge does not require tqdm